app/controllers/api/v1/members_controller.rb

```diff
@@ -0,0 +1,67 @@
+class Api::V1::MembersController < ApplicationController
+    include AuthenticationCheck
+  
+    before_action :is_user_logged_in
+    before_action :set_member, only: [:show, :update, :destroy]
+  
+    # GET /members
+    def index
+      @members = Member.all
+      render json: @members
+    end
+  
+    # GET /members/:id
+    def show
+      # your code goes here
+      # I'm not sure if set_member allows access to @member or if I am right in assigning it here first.
```

Comment:
A private method prevents calls from outside the class (except for [#send](https://www.rubydoc.info/stdlib/core/Object:send) and other meta-programming that would be best to avoid in general), but you're only referencing this `@member` within the same class, so you're fine regardless of whether the method is private or not.

If you wanted to access `@member` outside the class for whatever reason (you normally shouldn't do this for controllers), you'd use the `attr_*` methods like `attr_accessor :member` you learned in a previous lesson, and this is still independent of whether a method is private or not because an instance variable's visibility is only determined by whether you used the `attr_*` methods.

Either way, I'm not correct all the time, so it's still a good exercise to test things out yourself (I recommend with an automated test, because writing good tests is really important if you want your app to be robust long-term).